packages/ui/src/atom/circular-loader-atom/circular-loader-atom.vue

```diff
@@ -0,0 +1,135 @@
+<template>
+  <field-atom
+    :class="themeCssClass"
+    :style="sizeStyle"
+  >
+    <svg
+      class="circular-loader-atom-back"
+      :style="sizeStyle"
+    >
+      <circle
+        :r="circleAttrs.r"
+        :cx="circleAttrs.cx"
+        :cy="circleAttrs.cy"
+        :stroke-width="lineThickness"
+      />
+    </svg>
+    <svg
+      class="circular-loader-atom-over"
+      :style="{ ...sizeStyle, strokeDasharray }"
+    >
+      <circle
+        :r="circleAttrs.r"
+        :cx="circleAttrs.cx"
+        :cy="circleAttrs.cy"
+        :stroke="circleAttrs.stroke"
+        :stroke-width="lineThickness"
+      />
+    </svg>
+    <slot />
+  </field-atom>
+</template>
+
+<script>
+import ThemeMixin from '@/mixins/ThemeMixin';
+import FieldAtom from '@/atom/field-atom/field-atom';
+
+// For initial state progress must show `action` like something is processing.
+// But if progress have `0` value, circle will be static and '0%' title.
+// Soo default behavior is always show animated circle without label
+// and if progress more than `1`, show progress title
+const MIN_PROGRESS_VALUE = 1;
+const DEFAULT_VIEW_VALUE = 2;
+const MAX_VALUE = 100;
+
+export default {
+  name: 'VProgressCircle',
+
+  props: {
+    progress: {
+      type: [Number, String],
+      default: 0,
+    },
+
+    animated: {
+      type: Boolean,
+      default: true,
+    },
+
+    size: {
+      type: [Number, String],
+      default: 82,
```

Comment:
maybe constants for size? for avoid magic numbers

Reply:
It is not this case, when we should move number to constant